docs/index.bs

```diff
@@ -2600,9 +2600,9 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       : Input
       :: |registration|, a [=/service worker registration=]
-      :: *force bypass cache flag*, an optional flag unset by default
+      :: |force bypass cache flag|, an optional flag unset by default
```

Comment:
We don't tend to have spaces in variable names. Also, we tend to use booleans rather than flags in variables (last I heard, the plan is to move away from flags altogether, but I'm not sure what the status of that is).

So `:: |forceBypassCache|, an optional boolean, false by default`?